configs/oasisengine.json

```diff
@@ -2,28 +2,7 @@
   "index_name": "oasisengine",
   "start_urls": [
     {
-      "url": "https://oasisengine.cn/(?P<version>.*?)/docs/",
-      "variables": {
-        "version": [
-          "0.3",
-          "0.4",
-          "0.5",
-          "0.6"
-        ]
-      },
-      "page_rank": 2
-    },
-    {
-      "url": "https://oasisengine.cn/(?P<version>.*?)/api/",
-      "variables": {
-        "version": [
-          "0.3",
-          "0.4",
-          "0.5",
-          "0.6"
-        ]
-      },
-      "page_rank": 1
+      "url": "https://oasisengine.cn/(?P<version>.*?)/"
```

Comment:
I did not saw the `version` was still present there... that's the issue

Reply:
It works, thank you! @shortcuts I've seen the PR (https://github.com/algolia/docsearch-configs/pull/4877).

DocSearch is a wonderful product but it's difficult to debug.

Reply:
Don't worry, all of these process are fixed with our new infra: https://docsearch.algolia.com/docs/migrating-from-legacy

Keep an eye out on your emails :D